libs/designsystem/src/lib/components/dropdown/dropdown-popover.component.spec.ts

```diff
@@ -328,6 +328,26 @@ describe('DropdownComponent (popover version)', () => {
         });
       });
 
+      describe('and ArrowDown key is pressed', () => {
+        beforeEach(fakeAsync(() => {
+          spectator.dispatchKeyboardEvent(spectator.element, 'keydown', 'ArrowDown');
+          tick(openDelayInMs);
+        }));
```

Comment:
Any reason to do beforeEach here when there is only one `it`-block?

Reply:
No I don't see any reason 😅 I'll refactor it.